test/tests/auxkernels/diffusion_flux/tests

```diff
@@ -0,0 +1,7 @@
+[Tests]
+  [./test]
+    type = 'Exodiff'
+    input = 'diffusion_flux.i'
+    exodiff = 'diffusion_flux_out.e'
```

Comment:
Add this line to get it to pass everywhere:
```
abs_zero = 1e-9
```

Reply:
do you mean   nl_abs_tol = 1e-9?

Reply:
No.  Add that abs_zero to this file.  It sets the 'floor' used in the exodiff comparison, which makes it skip the check if a variable has a value below that floor.